app/views/publishers/vacancies/activity_log/show.html.slim

```diff
@@ -0,0 +1,17 @@
+- content_for :page_title_prefix, t(".page_title")
+
+= render "publishers/vacancies/banner", vacancy: vacancy, title: "Activity log"
+= render "publishers/vacancies/tabs_show", vacancy: vacancy
+
+.govuk-grid-row
+  .govuk-grid-column-two-thirds
+    - unless @versions.present?
+      p.govuk-body-m = t("publishers.activity_log.no_changes")
+    - @versions.each do |version|
+      - version.changeset.each_pair do |attribute, diff|
+        p.govuk-body class="govuk-!-font-weight-bold"
+          = vacancy_activity_log_item(attribute, diff.last, organisation_type_basic(vacancy.organisation))
+
+      p.govuk-body-s class="govuk-!-margin-top-1"
+        | By #{version.actor.given_name} #{version.actor.family_name} on #{version.created_at}
```

Comment:
I wonder if we should have a `papertrail_display_name` method on `Publisher` and `SupportUser` that we can use here instead of having given name and family name here in the view 🤔

For example:
```ruby
class Publisher
  def papertrail_display_name
    "#{given_name} #{family_name} (#{organisations.first.name})"
  end
end
```

and

```ruby
class SupportUser
  def papertrail_display_name
    "#{given_name} #{family_name} (DfE Teaching Vacancies team)"
  end
end
```